mikupad.html

```diff
@@ -3829,6 +3860,25 @@
 		showProbs && promptChunks[currentPromptChunk?.index]?.completion_probabilities?.[0]?.probs,
 		[promptChunks, currentPromptChunk, showProbs]);
 
+	const toggleSettings = () => {
+		toggleModal("settings");
+		document.getElementsByClassName("SelectBox")[0].style.display = modalState.settings ? "none" : "block";
+		document.getElementsByClassName("horz-separator")[0].style.display = modalState.settings ? "none" : "block";
+		for (const collapseGroup of document.getElementsByClassName("collapsible-group"))
+			collapseGroup.style.display = modalState.settings ? "none" : "block";
+	}
+	function updateSize() {
+		// Show sidebar on desktop if it was hidden just before
+		if (window.innerWidth > 767.98 && !modalState.settings) {
+			toggleSettings();
+		}
+	}
+	let timeout = false;
+	window.addEventListener('resize', () => {
+		// Without this delay, the cpu usage is high when resizing window and would lag for some reason (useful for debugging)
+		clearTimeout(timeout);
+		timeout = setTimeout(updateSize, 100);
+	});

```

Comment:
Afaict this will add an event listener every react render, you could fix this by using `useEffect` but I think a better way to solve this problem would be to make a new css style like `.mobile-hide { display: none; }` and put it inside the media query, then theorically the elements with this rule would be hidden only on mobile.